components/dropdown/dropdown-content-mixin.js

```diff
@@ -129,6 +151,10 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 			_bottomOverflow: {
 				type: Boolean
 			},
+			_closing: {
```

Comment:
I think this can actually have `attribute: false` and be called `_closing` since you're not actually using the attribute on the dropdown -- it's being bound to a `<div>` which actually has the attribute.